src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -482,6 +482,7 @@ struct SvgStyleProperty
     char* cssClass;
     bool paintOrder; //true if default (fill, stroke), false otherwise
     SvgStyleFlags flags;
+    SvgStyleFlags flagsImportance;
```

Comment:
It's not clear between flags vs flagsImportance. It would be nicer if you add a comment for a new flag purpose.

Reply:
added comment